### PR TITLE
alert-component

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please Review it**
Alert

### CODE REVIEW

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    height: 30px;
+    border-radius: 5px;
+color: grey;
+opacity: 0.9;
+
+}
+.alert-success{
+    height: 30px;
+    background-color: #17D7A0; 
+    border-radius: 5px;
```
Comment (nightlight-wander) [inline Components/Alert/alert.css:31]: you can use the color variable for colors ,to be defined in a seprate css file,which you can use across all components.

**Review by nightlight-wander — COMMENTED** [review]
The preview of alerts you attached looks good .Just a suggestion to maintain consistency otherwise it will create confusions later.

File: Components/Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+.alert{
+    width: 500px;
+    height: 10px;
+    background-color: burlywood;
+    margin: 50px 45px;
+    padding: 5px 10px 0px;
+}
+.alert-primary{
+    background-color: #A2D2FF;
```
Comment (zabihshaik) [inline Components/Alert/alert.css:15]: If the color is used repeatedly, better to use variables

**Review by zabihshaik — COMMENTED** [review]
Everything Looks Good.
Just 1 suggestion is that if the same colors are used across components, can use variables for them.

**Pull request merged**